build/common.sh

```diff
@@ -553,7 +553,14 @@ setup_kernel()
 {
 	echo ">>> Setting up kernel in ${1}"
 
-	tar -C ${1} -xpf ${SETSDIR}/kernel-*-${PRODUCT_ARCH}.txz
+	case ${PRODUCT_TARGET} in
+	arm|arm64)
```

Comment:
```suggestion
	arm*)
```

Reply:
Hmm, on second thought can we leave this out for a little bit? base needs name updates as well and invoking "tar" twice for the same thing in different paths is never. it also affects rename.sh and clean.sh targets

Reply:
Right, I also forgot about package extraction. Need to hold these product name append until everything related to it changed too.

Reply:
yes, for now please revert common.sh and kernel.sh -- the other changes to arm are perfectly fine

Reply:
common and kernel reverted